common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -517,6 +517,17 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  // Contributions
+  val ContributionsServerSideRenderingSwitch = Switch(
+    SwitchGroup.Feature,
+    "contributions-server-side-rendering-switch",
+    "If switched on, half of links to supoport.theguardian.com/contribute will have ?ssr=on ",
```

Comment:
**spelling:** `supoport` -> `support`

**bikeshedding:** this is a very specific description, is it possible the behaviour will change? 
`half of links` -> `a percentage of links` ?? YMMV 😄

Reply:
The only behaviour change will be this switches deletion sometime next week!